sections/queueing_model/proportion_within_target/main.tex

```diff
@@ -0,0 +1,141 @@
+            & \\
+        1 - (\mu C) ^ {\min(v,T) - C} \mu  
+            &  \\
+        \qquad \times \sum_{k=1}^{\mid \vec{r} \mid} \sum_{l=1}^{r_k} 
+        \frac{\Psi_{k,l}(-\lambda_k)t^{r_k - l} 
+        e^{-\lambda_k t}}{(r_k - l)! (l - 1)!}, 
+            & \textbf{if } C > 1 \\
+        \textbf{where } \vec{r}=(\min(v, T) - C, 1) 
+            & \textbf{and } v, T  > C\\
+        \hspace{1.15cm} \vec{\lambda}=(C \mu, \mu) \\
+        \hspace{1.15cm} \lambda_0 = 0, r_0 = 1 \\
+            & \\
+        1 - e^{-\mu t}, & \textbf{if } v \leq C \\ 
+            & \textbf{or } T \leq C \\
+    \end{cases}
+\end{equation}
+
+The function \(\Psi_{k,l}\) used in equations 
+(\ref{eq:proportion_within_target_type_1_from_state}) and 
+(\ref{eq:proportion_within_target_type_2_from_state}) is defined as:
+
+\begin{equation}
+    \Psi_{k,l}(t) = 
+    \begin{cases} 
+        \frac{(-1)^{l} (l-1)!}{\lambda_2} \left[\frac{1}{t^l} - \frac{1}
+        {(t + \lambda_2)^l}\right] , & k=1 \\
+        - \frac{1}{t (t + \lambda_1)^{r_1}}, & k=2
+    \end{cases} \nonumber \\
+    % & \textbf{and} \quad \qquad \lambda_0 = 0, r_0 = 1
+\end{equation}
+
+Refer to appendix \ref{sec:appendix_mean_proportion} for a more in-depth 
+explanation of the origins of equations 
+(\ref{eq:proportion_within_target_type_1}) - 
+(\ref{eq:proportion_within_target_type_2_from_state}).
+
+\subsubsection{Example}
+Figures \ref{fig:markov_vs_des_proportion_comparison_1},
+\ref{fig:markov_vs_des_proportion_comparison_2} and 
+\ref{fig:markov_vs_des_proportion_comparison_overall} show a comparison of the
+mean proportion of individuals within target when using Markov chains and 
+discrete event simulation (appendix~\ref{sec:appendix_des}).
+The simulation was ran 100 times and the recorded proportions at each iteration 
+is used to populate the violin plots.
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=.8\textwidth]{imgs/proportion_within_target_comparison/proportion_1.pdf}
+    \caption{
+        Comparison of proportion within target time for type 1 individuals 
+        between values obtained from the Markov chain formulas and values 
+        obtained from simulation. 
+    }
+    \label{fig:markov_vs_des_proportion_comparison_1}
+\end{figure}
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=.8\textwidth]{imgs/proportion_within_target_comparison/proportion_2.pdf}
+    \caption{
+        Comparison of proportion within target time for type 2 individuals 
+        between values obtained from the Markov chain formulas and values 
+        obtained from simulation. 
+    }
+    \label{fig:markov_vs_des_proportion_comparison_2}
+\end{figure}
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=.8\textwidth]{imgs/proportion_within_target_comparison/proportion_overall.pdf}
+    \caption{
+        Comparison of proportion within target time for both type of 
+        individuals between values obtained from the Markov chain formulas 
```

Comment:
Let us add 2 other versions of these.

- [ ] Include a table with 3 columns. The rows correspond to the parameters of the system (choose 3 relatively different systems). You might not need this and can just refer to the table in the waiting time section.
- [ ] Plot the proportion comparisons as 3 separate pdfs. This will lead to 3 subfigures for each figure. (3 plots next to each other)
- [ ] Include a bit more of a discussion in the captions (confirming obvious general trends as well as the fact that they match).